notificationservice.cpp

```diff
@@ -46,6 +46,7 @@ public slots:
                     else if(mReader.name() == "ai") appIcon = mReader.readElementText();
                     else if(mReader.name() == "su") summary = mReader.readElementText();
                     else if(mReader.name() == "bo") body = mReader.readElementText();
+                    else if(mReader.name() == "vb") vibrate = mReader.readElementText();
```

Comment:
Well even the silent one is a visual alert. I think vibration is the key one the user is interested in.